src/app/[lng]/swap/[segments]/page.tsx

```diff
@@ -0,0 +1,80 @@
+import { siteName } from '@/app/lib/metadata';
+import { getSiteUrl } from '@/const/urls';
+import { getChainsQuery } from '@/hooks/useChains';
+import { getTokensQuery } from '@/hooks/useTokens';
+import { getChainByName } from '@/utils/tokenAndChain';
+import type { Metadata } from 'next';
+import { notFound } from 'next/navigation';
+import SwapPage from 'src/app/ui/swap/SwapPage';
+
+export async function generateMetadata({
+  params,
+}: {
+  params: { segments: string };
+}): Promise<Metadata> {
+  const { chains } = await getChainsQuery();
+  const sourceChain = getChainByName(chains, params.segments);
+  const title = `Jumper | How To Swap on ${sourceChain?.name} | A Complete Guide`;
+
+  const openGraph: Metadata['openGraph'] = {
+    title: title,
+    description: `Jumper offers the best way to swap tokenA to tokenB on ${sourceChain?.name} with the fastest speeds, lowest costs, and most secure swap providers available.`,
```

Comment:
should be "to swap tokens on"

Reply:
Fixed, it is now:
`Jumper offers the best way to swap tokens on ${sourceChain?.name} with the fastest speeds, lowest costs, and most secure swap providers available.`